scripts/morty/analysis.py

```diff
@@ -0,0 +1,273 @@
+import solvers
+import numpy as np
+from data import DataHandler
+from scipy import integrate
+
+
+class AnalysisCollection:
+    def __init__(self, analysis_params, run_params, data_params):
+        """
+        This class contains a collection of analysis procedures which
+        can be run.
+        The data is returned in an easily-plottable dictionary as
+        well as saved to
+        text files for future analysis.
+
+        Parameters
+        ----------
+        analysis_params : dict
+            key : str
+                Name of analysis parameter
```

Comment:
This is a tautology. For clarity, maybe give an example. How is an analysis parameter different from a run parameter or a data parameter? Also, it would be best to also define the values expected, not just the keys. What type are they? Any type?